app/Http/Controllers/AdminController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use DB;
+
+class AdminController extends Controller
+{
+    public function showChangeForm()
+    {
+        return view('admin.changeAdmin');
+    }
+
+    public function changeAdmin(Request $request)
+    {
+        $request->validate([
+            'email' => 'required',
```

Comment:
shouldn't be username instead of mail? anyway, you have to validate it before updating